apps/src/Sound.js

```diff
@@ -339,22 +340,61 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
+  try {
+    if (!window.Audio) {
+      return false;
+    }
+
+    var audioTest = new window.Audio();
+    if (
+      this.config.hasOwnProperty('bytes') &&
+      audioTest.canPlayType('audio/mp3')
+    ) {
+      return this.config.bytes;
+    }
+  } catch (e) {
+    console.warn('No bytes provided or mp3 is not supported');
+  }
+
+  return false;
+};
+
 Sound.prototype.preload = function() {
```

Comment:
It looks like there are two separate code paths in preload. Consider migrating this to `Sound.prototype.preloadFile` and `Sound.prototype.preloadBytes`